tests/upgrade/test_upgrade_migration.py

```
@@ -0,0 +1,129 @@
+        pytest.param(
+            py_config["tests_params"]["test_upgrade_cold_migration"],
+        )
+    ],
+    indirect=True,
+    ids=["upgrade-cold"],
+)
+@pytest.mark.usefixtures("cleanup_migrated_vms")
+class TestUpgradeColdMigration:
+    """Cold migration with MTV operator upgrade between plan creation and execution."""
+
+    def test_upgrade_mtv(self, upgrade_script_path: str, upgrade_plan_resource: Plan) -> None:
```

Ruff (0.15.13)

[warning] Unused method argument: `upgrade_plan_resource`

(ARG002)